### PR TITLE
Moved Mr. Sandman from netobserv to main branch

### PR DESCRIPTION
**Pull request by nathan-weinberg**
This script, formerly only used by NetObserv QE, is now being used by PerfScale QE as well. As such, @paigerube14 and I discussed moving it to the `main` branch to make collaboration easier.

### CODE REVIEW

File: scripts/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+argparse>=1.4.0
```
Comment (paigerube14) [inline scripts/requirements.txt:1]: think we also need datetime

Comment (paigerube14) [reply scripts/requirements.txt:1]: nvm, ran without needing to install

**Review by paigerube14 — APPROVED** [review]
/lgtm

**Comment by paigerube14** [comment]
@nathan-weinberg not sure why cluster density v2 on rosa is a required test here. Need to look into that. think it'll block the merging...

**Comment by nathan-weinberg** [comment]
@paigerube14 @svetsa-rh no worries! no rush on this - let me know if you need anything from my side!

**Comment by svetsa-rh** [comment]
@nathan-weinberg No action needed from your side for now. @paigerube14 is going to remove one of the files and merge changes. If the above test fails you can go ahead and rebase and pull her changes to skip this test.

**Comment by paigerube14** [comment]
The one enhancement I had mentioned to Nathan is to add a bash file that has all of the items in the dict set up as environment variables so we'll be able to export the file and not need to parse out every variable to set it. Will be a follow up PR for that

**Pull request merged**